Tribler/Test/Core/test_leveldb_store.py

```diff
@@ -48,14 +48,44 @@
 V = "bar"
 
 
-class ClockedLevelDbStore(LevelDbStore):
+class ClockedAbstractLevelDBStore(LevelDbStore):
     _reactor = Clock()
 
 
-class TestLevelDbStore(BaseTestCase):
+class ClockedLevelDBStore(ClockedAbstractLevelDBStore):
+    from leveldb import LevelDB
+    _leveldb = LevelDB
+
+    def __init__(self, store_dir):
```

Comment:
You can drop __init__  and get_write_batch if you set _writebatch as a static attribute like _leveldb is.
Also keep both modules and let the plyvel one import what's needed from the other one.